forage-android/src/main/java/com/joinforage/forage/android/core/ui/element/ForagePinElement.kt

```diff
@@ -135,6 +138,9 @@ abstract class ForagePinElement @JvmOverloads constructor(
         level = DeprecationLevel.WARNING,
         replaceWith = ReplaceWith("")
     )
+    /**
+     * @deprecated setHintTextColor (for **PIN** elements) is deprecated.
+     */
```

Comment:
included these `@deprecated` comments for the Dokka reference docs, the `@Deprecated` (title case) decorator/annotation is for showing an error in the editor!


<img width="755" alt="image" src="https://github.com/teamforage/forage-android-sdk/assets/32694765/9b69c033-701f-4f30-ba7a-12c563cbc8c5">